src/invidious.cr

```diff
@@ -135,6 +135,9 @@ end
 OUTPUT = CONFIG.output.upcase == "STDOUT" ? STDOUT : File.open(CONFIG.output, mode: "a")
 LOGGER = Invidious::LogHandler.new(OUTPUT, CONFIG.log_level)
 
+# Check pending migrations
+Invidious::Database::Migrator.new(PG_DB).check_pending_migrations
+
 # Check table integrity
 Invidious::Database.check_integrity(CONFIG)
```

Comment:
I noticed this method while working... and this is a bit of a monster hidden behind a simple name. It performs database updates besides doing checks. This might have to be taken out completely once the migration files are for sure the way forward since it reads the sql files.

Reply:
Yeah, this is a remnant I haven't dare to touch. I think that it should be safe to remove all of that code.